src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-## Colors
+Use an **inverse paragraph** if you need to put a paragraph on top of a dark background.
+
+```jsx
+<Paragraph inverse>
+	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
+	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
+</Paragraph>
+```
 
-### Paragraph with custom color
+For both inverse and default type colours, you can also use a dark and light variant. Use sparingly when needed to provide an additional level of hierarchy or emphasis.
 
 ```jsx
-<Paragraph color="accent">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="medium">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="light">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="danger">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
-<Paragraph color="success">Aenean auctor justo nisl, in ullamcorper turpis pharetra mattis. Duis eget vulputate est. Nullam a velit pharetra, hendrerit magna vitae, accumsan magna.</Paragraph>
+<Paragraph light>
+	Light
+</Paragraph>
+
+<Paragraph>
+	Default
+</Paragraph>
+
+<Paragraph dark>
+	Dark
+</Paragraph>
+
+<Paragraph inverse light>
+	Inverse light
+</Paragraph>
+
+<Paragraph inverse>
+	Inverse
+</Paragraph>
+
+<Paragraph inverse dark>
+	Inverse dark
+</Paragraph>
 ```
+
+
+## Voice & Tone
+
+Aim to keep paragraphs as **short as possible**. If a paragraph is longer than five sentences, break it into two.
+
+When writing long passages of text comprised of many paragraphs, highlight important information using **strong** and _emphasis_ tags. Use headings and lists to make content easier to scan.
+
+## Accessibility
+
+- Ensure text colour has a contrast ratio of at least 7:1 (WCAG AAA).
+- Use relative units to ensure that text can be resized by users.
+- Line height should be at least a space-and-a-half within paragraphs. (1.5)
+- Paragraph spacing should be at least 1.5 times the line height.
+- Don't allow line lengths to span more than 80 characters.
+- Font size should be a minimum of 16px.
```

Comment:
Even for `small` paragraphs? Might be worth clarifying (I feel like `14px` "small" paragraphs are okay).

Reply:
The default theme puts the small paragraphs at 16px, but I'm okay dialling this back a bit since I think the spec only says that type needs to be resizeable up to 200%.